st2api/st2api/controllers/resource.py

```diff
@@ -50,7 +50,9 @@ class ResourceController(rest.RestController):
     from_model_kwargs = {}
 
     # Maximum value of limit which can be specified by user
-    max_limit = cfg.CONF.api.max_page_size
+    @property
+    def maxlimit(self):
```

Comment:
Another option also is lint ignore statement, but not 100% this will work since it probably just won't report error but it will still parse the line causing the error...